attributes/default.rb

```diff
@@ -26,6 +26,13 @@
 # Create an application key on the Account Settings page
 default['datadog']['application_key'] = nil
 
+# Use this attribute to send data to multiple accounts
```

Comment:
nitpick: `multiple` -> `additional`? Just want to make extra-clear that these attributes are used to configure accounts on top of the "default" one.

Also, do you think it's worth mentioning that this affects the Agent _and_ the datadog handler>